test123.js

```diff
@@ -1 +1,10 @@
+// the hello world program
 console.log('Hello World');
+
+console.log('Hello Universe');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=3437)
```suggestion
console.log("Hello Universe");
```